mibidata/runs.py

```diff
@@ -11,7 +11,7 @@
 MASS_CALIBRATION_PARAMETERS = ('TimeResolution', 'MassGain', 'MassOffset',
                                'XSize', 'YSize')
 FOV_PATTERN = re.compile('^(Depth_Profile|Chemical_Image)$')
-
+_MOTOR_STEP = 0.1  # motor step in microns
```

Comment:
Can you be even more explicit here -- there are 0.1 motor steps per micron, or 0.1 microns per motor step?

Reply:
done